### PR TITLE
Formula for installing and enabling prometheus node_exporter

### PR DESCRIPTION
**Pull request by renner**
This patch adds an initial version of a formula to install and enable the Prometheus `node_exporter`.

### CODE REVIEW

File: prometheus-exporters-formula/prometheus-exporters/map.jinja [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{% set exporters = salt['grains.filter_by']({
+    'RedHat': {
+        'node_exporter_package': 'golang-github-prometheus-node_exporter',
+        'node_exporter_service': 'prometheus-node_exporter',
+    },
+    'Suse': {
+        'node_exporter_package': 'golang-github-prometheus-node_exporter',
+        'node_exporter_service': 'prometheus-node_exporter',
+    },
```
Comment (mcalmer) [inline prometheus-exporters-formula/prometheus-exporters/map.jinja]: No Ubuntu?

Comment (cavalheiro) [reply prometheus-exporters-formula/prometheus-exporters/map.jinja]: Since this is just an MVP, not sure if we need to enable Ubuntu right away.

Comment (renner) [reply prometheus-exporters-formula/prometheus-exporters/map.jinja]: We can add `Ubuntu`, it's easy, we just need to check what is the name of the package and service. Also `RedHat` at the moment is just a copy of the `Suse` names, will check if package or service name actually differs.

**Comment by renner** [comment]
@mcalmer: The cleanup can be achieved via `init.sls` by setting `node_exporter_enabled` to `False` accordingly. For now I would suggest to see if this is maybe sufficient, otherwise we can add a separate state for performing the cleanup later.

File: prometheus-exporters-formula/prometheus-exporters/map.jinja (hunk 1 above)
Comment (renner) [reply prometheus-exporters-formula/prometheus-exporters/map.jinja]: Added support for Ubuntu (untested), the package should be available from Universe. Dropping support for RedHat for now as I could not easily find the package and service names.

**Pull request merged**